tests/acceptance/features/webUISharingInternalGroupsSharingIndicator/shareWithGroups.feature

```diff
@@ -38,7 +38,7 @@ Feature: Sharing files and folders with internal groups
       | fileName   | expectedIndicators |
       | inside.txt | user-indirect      |
 
-  @issue-2060 @issue-6894
+  @skipOnOCIS @issue-2060 @issue-6894
```

Comment:
Can we add a similar comment here and in other scenarios too so that we won't be confused in future why this is skipped in ocis? https://github.com/owncloud/web/blob/09120268c6bc311f3456bb6aef70b11e97618f51/tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature#L15-L16

Reply:
sure thanks for reporting!!